spec/spec_helper.rb

```diff
@@ -49,13 +48,9 @@
 
 if defined?(::ActiveRecord)
   require 'support/active_record'
-elsif defined?(::Mongoid)
-  require 'support/mongoid'
-elsif defined?(::Sequel)
-  require 'support/sequel'
 else
   RSpec.configure do |config|
-    %i[orm mongoid active_record sequel].each do |group|
+    %i[orm active_record].each do |group|
```

Comment:
Do we really need to be able to run the specs w/o `ActiveRecord`?

Also, L46 could be removed.